examples/Training/python/Create_Image.py

```diff
@@ -15,13 +15,17 @@
 from Parse_OMERO_Properties import USERNAME, PASSWORD, HOST, PORT
 from Parse_OMERO_Properties import imageId
 
-# Create a connection
+"""
+start-code
+"""
+
+# Create a connection:
 # =================================================================
```

Comment:
@ximenesuk: colon must have been added for unification before I release we did not need one, and forgot to go back to the addition.
The underline can be adjusted now that I fixed the script. It was not the case before.